scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
+            /* hotkey[0] => Category, hotkey[1] => Description, hotkey[2] => Shortcut, hotkey[3] => Options */
+
+            this.bind(hotkey[2], function (e) {
```

Comment:
does it ever unbind those shortcuts? because if you trigger it multiple times (like going from workspace to settings and back) it might do it again and it would leak memory